docs/Makefile

```diff
@@ -87,6 +87,9 @@ redirects: setup
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/dirhtml."
 
+.PHONY: configyaml
+configyaml:
```

Comment:
nit, could add a rule like:

```make
_data/db_config.yaml: ../db/config.cc
    python _utils/parse_db_properties.py "../db/config.cc" "_data/db_config.yaml"

configyaml:  _data/db_config.yaml
```

Reply:
I find it a bit more difficult to follow, if it's not a blocker to merge the PR I'd prefer leaving it as is.

Reply:
no, it's not. the whole idea is just to avoid regenerating the output file if it is already updated.

Reply:
As of now, the command it doesn't run automatically with every documentation build. It needs to be triggered manually.